app/src/main/java/xyz/whereuat/whereuat/ContactCard.java

```diff
@@ -1,21 +1,30 @@
 package xyz.whereuat.whereuat;
 
 import android.content.Context;
-import android.location.Location;
+import android.database.Cursor;
 import android.util.AttributeSet;
 import android.util.Log;
 import android.view.View;
+import android.widget.Toast;
 import android.widget.ViewFlipper;
 
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+
+import xyz.whereuat.whereuat.db.command.QueryCommand;
+import xyz.whereuat.whereuat.db.entry.ContactEntry;
+
 /**
  * This class is a wrapper around a ViewFlipper to provide default functionality for a contact card
  * in the contacts GridView. It sets the ViewFlipper to be a square and sets its click listeners.
  */
 public class ContactCard extends ViewFlipper {
     private static final String TAG = "ContactCard";
+    private Context mContext;
```

Comment:
Does anything else in the class use it? Why not have it as a local variable of the constructor?